### ISSUE TITLE
Alter background for page headings

### ISSUE THREAD

**Issue by reveurguy** [kubernetes/website#38921]
**This is a Feature Request**
I have some ideas for changes/improvements to the website design.

**What would you like to be added**

<img width="1073" alt="Screenshot 2023-01-13 at 9 30 41 PM" src="https://user-images.githubusercontent.com/59417546/212364352-bf5ef7c3-4a4e-49e6-9877-553805d652c4.png">

- Changing the hero section, this is just an idea for a background for visual improvement. 
- Adding the project logo and adding some text information about the project ( 2-3 lines ) to the hero-section would make a good viewpoint when the website loads.
- Also along with the 'learn k8s basics' button, we can add other important links as well like github, slack etc. Currently those buttons are at the bottom of the page, adding them at the top as well can make it easier to access important links.

<img width="465" alt="Screenshot 2023-01-13 at 9 38 59 PM" src="https://user-images.githubusercontent.com/59417546/212366004-46ee7023-a9df-4425-802b-569fb9c9704d.png">
- another idea, adding a card like structure to some sections would improve the visualisation and provide structure as well.

#### Other ideas
Some more changes i can think of are:
- improving the video-section for better visibility and look
- adding card format like features(above) to case studies and  'we are a graduated project' section
- improving the kubeweekly subscribe section to utilise space and look better, we can also add a background like the hero-title section to this section as well, that can make it look better.

**Why is this needed**
- better usability of the website
- improved design and look of the website
- better utilisation of space and providing structure to sections

**Comments**

**Comment by NitishKumar06** [comment]
Here's my opinion on this issue:
- There's no need to add something to the hero section (or modify anything until you come across broken links/ bad redirecting link) because it's already working fine. However, the blue color looks good.
- I don't think there's need to add a separate link mentioning about slack, GitHub because these links are already visible at the end of the page. Even if you go through the [CNCF website](https://www.cncf.io/), it also follows the same approach for mentioning a link at the end of the introduction page.
- For adding a card like structure, it _may or may not_ help in solving this [issue](https://github.com/kubernetes/website/issues/34355#event-8084145917)

**Comment by sftim** [comment]
I like the card-like structure idea. How about splitting that into its own issue?

**Comment by reveurguy** [comment]
> Here's my opinion on this issue:
> 
> * There's no need to add something to the hero section (or modify anything until you come across broken links/ bad redirecting link) because it's already working fine. However, the blue color looks good.
> * I don't think there's need to add a separate link mentioning about slack, GitHub because these links are already visible at the end of the page. Even if you go through the [CNCF website](https://www.cncf.io/), it also follows the same approach for mentioning a link at the end of the introduction page.
> * For adding a card like structure, it _may or may not_ help in solving this [issue](https://github.com/kubernetes/website/issues/34355#event-8084145917)

ok we can let the hero section remain as it is.
Yeah the blue color looks good. We can add the blue background to footer as well, that will maintain consistency.
For the card structure, i couldn't understand the reference to the issue you mentioned.

**Comment by NitishKumar06** [comment]
> > Here's my opinion on this issue:
> > 
> > * There's no need to add something to the hero section (or modify anything until you come across broken links/ bad redirecting link) because it's already working fine. However, the blue color looks good.
> > * I don't think there's need to add a separate link mentioning about slack, GitHub because these links are already visible at the end of the page. Even if you go through the [CNCF website](https://www.cncf.io/), it also follows the same approach for mentioning a link at the end of the introduction page.
> > * For adding a card like structure, it _may or may not_ help in solving this [issue](https://github.com/kubernetes/website/issues/34355#event-8084145917)
> 
> ok we can let the hero section remain as it is. Yeah the blue color looks good. We can add the blue background to footer as well, that will maintain consistency. For the card structure, i couldn't understand the reference to the issue you mentioned.

Go with the suggestions of @sftim and break down it into issues.

Mentioning about [this](https://github.com/kubernetes/website/issues/34355#event-8084145917), the idea in that issue is to create a page to list some blogs. It'd be better to implement (something like a card structure) once that page has been created. Long way to go before that I guess.

**Comment by reveurguy** [comment]
> > > Here's my opinion on this issue:
> > > 
> > > * There's no need to add something to the hero section (or modify anything until you come across broken links/ bad redirecting link) because it's already working fine. However, the blue color looks good.
> > > * I don't think there's need to add a separate link mentioning about slack, GitHub because these links are already visible at the end of the page. Even if you go through the [CNCF website](https://www.cncf.io/), it also follows the same approach for mentioning a link at the end of the introduction page.
> > > * For adding a card like structure, it _may or may not_ help in solving this [issue](https://github.com/kubernetes/website/issues/34355#event-8084145917)
> > 
> > 
> > ok we can let the hero section remain as it is. Yeah the blue color looks good. We can add the blue background to footer as well, that will maintain consistency. For the card structure, i couldn't understand the reference to the issue you mentioned.
> 
> Go with the suggestions of @sftim and break down it into issues.
> 
> Mentioning about [this](https://github.com/kubernetes/website/issues/34355#event-8084145917), the idea in that issue is to create a page to list some blogs. It'd be better to implement (something like a card structure) once that page has been created. Long way to go before that I guess.

ok yeah i'll make new issues.
for the card structure, i think we can implement that on the homepage initially and on some other pages like community etc after that and then we can add them to the blogs page whenever it is created. while working on that, i'll create a css class for the card structure which we can then simply add to the sections required along with some modification if needed.

**Comment by NitishKumar06** [comment]
Yes, please go ahead! Let's see what you bring to the discussion table:)

**Comment by reveurguy** [comment]
I have made a PR for the background changes.

**Comment by reveurguy** [comment]
Apart from the background and card structure changes, i had mentioned a couple of other ideas as well.

1. improving the video section. As shown in the ss below, the heading is not clearly visible and readable. Also the button size is too large which overpowers the text, ideally the buttons work as call-to-action and should be smaller in size comparable to the heading and text. So that's one thing we should sort out.
**-->** The solution to the visibility part can be making some design changes to the section, like adding a shadow behind the text( or dimming the image) or choosing another image which is less bright and then making changes to the heading and buttons accordingly.
<img width="1263" alt="Screenshot 2023-01-17 at 5 16 46 PM" src="https://user-images.githubusercontent.com/59417546/212891070-4aa3377d-1e20-477f-b9ad-c6f3acf3625b.png">

2.
- increasing the 'we are a cncf graduated project' part for better emphasis on that particular part ( just a small change suggestion)
- improving the sign-up for kubeweekly section. We can add a background like the one in hero part or something similar to the section and changing the button, input box size along with minor style changes.

<img width="1247" alt="Screenshot 2023-01-17 at 5 23 33 PM" src="https://user-images.githubusercontent.com/59417546/212892323-bb5435c4-abfc-45d8-b2f7-d6549f67258c.png">


After working on the homepage, improvements can be made on other pages as well like community, case studies, partners etc. Improvements like adding card structure where suitable, making style and design changes. These can be discussed after the current changes are worked upon.

**Comment by reveurguy** [comment]
Also regarding the background changes, i have received positive feedback for that so we can discuss upon that as well if any changes are required for that.

**Comment by sftim** [comment]
I'd still recommend splitting out the ideas into _more than one issue_. Then, we (SIG Docs) can triage and accept the ideas that folks are happy with, and also different people can work on implementations in parallel.

**Comment by reveurguy** [comment]
Ok. Let's keep this issue as the background change. I'll create new issues for the other ideas.

**Comment by sftim** [comment]
I'm happy with the existing pattern that we use as a background for headings (when there is no banner active). Why should we change it?

**Comment by sftim** [comment]
@reveurguy, if you can explain the reason to change, that'll help people work out what design to adopt.
